### ISSUE TITLE
All 17 Species Files checklists in Oct-Nov 2022

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/testing#210]
**Updates imported to CLB 2022-09-14**

**Comment by yroskov** [comment]
FIRST BATCH


-- | GSD name | Imported, spp | Sector | Metadata | Classification | Sync | Report
-- |-- | -- | -- | -- | -- | -- | -- 
[ ] | SF Aphid | 5960 | infraorder Aphidomorpha | 2022-10- | OK | 2022-10- | --
[ ] | SF Chrysididae | 199 | family Chrysididae | 2022-10- | OK |  2022-10- | 
[ ] | SF Coleorrhyncha | (115) | paleofamilies Hoploridiidae, Karabasiidae & Progonocimicidae | 2022-10- | OK | 
[ ] | SF Coreoidea | 3120 | superfamily Coreoidea | 2022-10- | OK | 2022-10- | 
[ ] | SF Lygaeoidea | 4683 | superfamily Lygaeoidea | 2022-10- | OK | 2022-10- | 
[ ] | SF Mantodea | 2537 | order Mantodea | 2022-10- | OK | 2022-10- | 
[_] | SF Orthoptera | 29135 | order Orthoptera | 2022-10- | OK | 2022-10- | 
[ ] | SF Phasmida | 3415 | order Phasmida | 2022-10- | suborder Agathemerodea is empty, blocked in CoL | 2022-10- |
[ ] | SF Isoptera | 3062 | infraorder Isoptera | 2021-10- |  | 2021-10- |

**Comment by yroskov** [comment]
- [ ] We are not able to update metadata: _Request failed with status code 400
Dataset 1021 is registered in GBIF af66d4cf-0fd2-434b-9334-9806a5efa6f7 and updates are not allowed_
= for attention of @mdoering

**Comment by mdoering** [comment]
Yes, that is a new security "feature" that prevends manual changes in metadata for datasets that are registered in GBIF.
There is still an ongoing discussion here that you might want to get involved with: https://github.com/CatalogueOfLife/checklistbank/issues/1088

The desire is to keep the metadata for the same dataset +/- identical in GBIF & CLB. And also for external datasets that come with some metadata which would otherwise overrite manual edits - with the exception of the new metadata merges, see https://github.com/CatalogueOfLife/backend/issues/1047.

**Comment by mdoering** [comment]
Dont those SF archives come with metadata already?

**Comment by yroskov** [comment]
- [ ] Superspecies: accepted names with portion **supersp** reported in ISSUES Inconsistent Name (e.g. Eosembia supersp aequicercata Ross, 2007). These taxa blocked in CLB via decisions in current design.

TASKS report **ACC-SYN sp, diff acc, same auth** include binomial species as _synonym_ to superspecies and as an accepted name. Synonyms blocked in CLB via decisions in current design.

**Comment by yroskov** [comment]
- [ ] Interpretation of Nomenclatural Status in SFs checklists: "potentially valid" instead of "valid" https://github.com/CatalogueOfLife/backend/issues/1181

Where the problem sits (SF - TW - COLDP - CLB)?

**Comment by mdoering** [comment]
CLB. I tried to explain that it is behaving as expected though.

**Comment by yroskov** [comment]
Thank you, @mdoering. The problem is, status "potentially valid" with all valid SF species in the CoL makes SF authors uncomfortable.

**Comment by yroskov** [comment]
Updates once per year (in Jan):

Coleorrhyncha 
Embioptera (after May 2024)
Mantophasmatodea 
Zoraptera
 
Monthly updates: 

Dermaptera 
~~Embioptera~~ 
Isoptera 
Plecoptera
Psocodea 

Heidi, 2024-05-01: _I am going to move Embioptera over to the annual update list_
